site/contents.js

```diff
@@ -48,7 +48,16 @@ const contents = [
   {
     group: 'integrations',
     label: 'Integrations',
-    pages: ['vite', 'esbuild', 'webpack', 'next', 'parcel', 'rollup', 'gatsby'],
+    pages: [
+      'vite',
+      'esbuild',
+      'webpack',
+      'next',
+      'parcel',
+      'remix',
+      'rollup',
+      'gatsby',
```

Comment:
Order is a bit random at the moment. I think we should keep them alphabetical, to avoid any bias.

```suggestion
      'esbuild',
      'gatsby',
      'next',
      'parcel',
      'remix',
      'rollup',
      'vite',
      'webpack',
```